<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageProcessClass</class>
 <widget class="QMainWindow" name="ImageProcessClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>642</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>820</width>
    <height>642</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>820</width>
    <height>642</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ImageProcess</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>581</height>
     </rect>
    </property>
    <property name="title">
     <string>Control Panel</string>
    </property>
    <widget class="QPushButton" name="pushButtonPhotograph">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Photograph</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonLuminescence">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Luminescence</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSubstract">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Substract</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSmooth">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Smooth</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonFusion">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Fusion</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSave">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>430</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonQuit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>530</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Quit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonClear">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>480</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="PhotographLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="LuminescenceLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="SubstractLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="SmoothLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QVTKWidget" name="qvtkWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>560</width>
      <height>560</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>120</width>
      <height>581</height>
     </rect>
    </property>
    <property name="title">
     <string>Colorbar</string>
    </property>
    <widget class="QSpinBox" name="spinBoxHighValue">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxLowValue">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>550</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>65534</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="imageprocess.qrc"/>
 </resources>
 <connections/>
</ui>
